components/brave_wallet/browser/solana_transaction.cc

```diff
@@ -224,16 +224,11 @@ SolanaTransaction::GetSignedTransactionBytes(
   std::vector<uint8_t> transaction_bytes;
 
   // Add message
-  auto message_bytes = message_.Serialize(nullptr);
-  if (!message_bytes)
-    return absl::nullopt;
-
-  // Get number of signers from message header
-  // absl::optional<std::uint8_t> num_signers = message_.GetNumberOfSigners();
-  uint8_t num_signers = (*message_bytes)[0];
-  if (!num_signers || num_signers != 1) {
+  std::vector<std::string> signers;
+  auto message_bytes = message_.Serialize(&signers);
+  // if ((!message_bytes) || (!(signers.size() == 1)))
+  if (!message_bytes || !(signers.size() == 1))
```

Comment:
nit: `if (!message_bytes || signers.size() != 1)` and remove the leftover comment at line 229.